torch_xla/csrc/aten_autograd_ops.cpp

```diff
@@ -21,6 +22,12 @@ namespace aten_autograd_ops {
 torch::Tensor EinsumAutogradFunction::forward(
     torch::autograd::AutogradContext* ctx, const c10::string_view equation,
     at::TensorList tensors) {
+  // We might get here after the autograd graph was created, but before
+  // python gets a chance to run. Give python a chance to run.
+  if (tensors[0].key_set().has(c10::DispatchKey::Python)) {
```

Comment:
@bdhirsh do we need to apply same thing to backward? Just curious why don't we do it in https://github.com/pytorch/xla/blob/master/torch_xla/csrc/aten_xla_type.cpp#L1076 instead of here.

Reply:
Hmm I don't... think we need to do. Empirically, it sounds like we don't from Shunting's testing.